examples/medplum-patient-intake-demo/data/core/valuesets.json

```
@@ -0,0 +1,182 @@
+{
+  "resourceType": "Bundle",
+  "type": "transaction",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:97139871-af48-46af-a8b4-ed16fa8b6b6f",
+      "request": {
+        "method": "PUT",
+        "url": "ValueSet?name=pregnancy-status-codes"
+      },
+      "resource": {
+        "resourceType": "ValueSet",
+        "status": "active",
+        "url": "http://example.com/pregnancy-status",
+        "name": "pregnancy-status-codes",
+        "title": "Pregnancy Status Codes",
+        "description": "These codes are used to indicate pregnancy status.",
+        "compose": {
+          "include": [
+            {
+              "system": "http://snomed.info/sct",
+              "concept": [
+                {
+                  "code": "146799005",
+                  "display": "Possible pregnancy"
+                },
+                {
+                  "code": "60001007",
+                  "display": "Not pregnant (finding)"
+                },
+                {
+                  "code": "77386006",
+                  "display": "Pregnancy"
+                }
+              ]
+            },
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v3-NullFlavor",
+              "concept": [
+                {
+                  "code": "UNK",
+                  "display": "Unknown"
+                }
+              ]
+            }
+          ]
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:6abaa6c8-d822-46a4-b3a2-70639475e5ae",
+      "request": {
+        "method": "POST",
+        "url": "ValueSet?name=us-core-usps-state",
```

```suggestion
        "url": "ValueSet",
```
When doing a POST, the query parameters are ignored. 


https://www.medplum.com/docs/fhir-datastore/fhir-batch-requests#conditional-batch-actions